src/deploy/osp_deployer/settings/sample-fx.properties

```diff
@@ -1,197 +0,0 @@
-[
```

Comment:
Would you mind moving the removal of this file into the PR for removing FX support?

Reply:
done